ds3-sdk/src/main/java/com/spectralogic/ds3client/commands/DeleteObjectsRequest.java

```diff
@@ -44,25 +49,20 @@
     // Constructor
     
     
-    public DeleteObjectsRequest(final String bucketName, final List<String> objects) {
+    public DeleteObjectsRequest( final String bucketName, final Map< String, Set< UUID > > objects )
+    {
         this.bucketName = bucketName;
         this.objects = objects;
         
         this.getQueryParams().put("delete", null);
     }
-
     
-    public DeleteObjectsRequest(final String bucketName, final Iterable<Contents> objs) {
-        this.bucketName = bucketName;
-        
-        this.getQueryParams().put("delete", null);
-        this.objects = contentsToString(objs);
-    }
-
-    private static List<String> contentsToString(final Iterable<Contents> objs) {
-        final List<String> objKeyList = new ArrayList<>();
+    
+    private static Map< String, Set< UUID > > contentsToString( final Iterable< Contents > objs )
```

Comment:
You deleted the constructor this code was used in.  I believe this code is now dead.  If so, it should be removed.

Reply:
I wasn't sure, removed now.